### ISSUE TITLE
Tempest is missing Weapon attributes

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#1324]
```
ship "Tempest"
	sprite ship/tempest
	...
		"weapon capacity" 271
		"engine capacity" 122
		(weapon would be here, but it is not)
	outfits
		"Sunbeam" 2
		...
```

**Comment by endless-sky** [comment]
Good catch. Fixed in cc7a97e.

**Comment by Amazinite** [comment]
Closing because it was never in a release.
